README.md

```diff
@@ -157,12 +157,13 @@ End-User WG|Every other week on Thursday at 10:00 PT|[Google Doc](https://docs.g
 eBPF|Every week on Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
 Prometheus Interoperability WG|Every other week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
-Client Instrumentation|Every Tuesday at 9:00 AM PT (primary) and every Monday at 12:30 PM PT (half hour)|[Google Doc](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w/edit)|[Slack](https://cloud-native.slack.com/archives/C0239SYARD2)|
+Client Instrumentation|Every Tuesday at 9:00 AM PT|[Google Doc](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w/edit)|[Slack](https://cloud-native.slack.com/archives/C0239SYARD2)|
 Kubernetes Operator|Every other week on Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4)|[Slack](https://cloud-native.slack.com/archives/C033BJ8BASU)|
 Community Demo Application|Every Monday at 8:30 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
 Functions as a Service (FAAS) |Every other Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
 OpenTelemetry Profiling WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
 Security | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
+Event WG|Every Friday at 10:00 AM PT|TBD Google Doc|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

Comment:
```suggestion
Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A/edit#heading=h.1nz72gv5evo2)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

Reply:
```suggestion
Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

Reply:
@jack-berg please make sure to pass on ownership to the document to the GC Google account as described here: https://github.com/open-telemetry/community/issues/1147#issuecomment-1234759420

Reply:
Done, thanks!

Reply:
(accepted ownership)